README.md

```diff
@@ -1,6 +1,43 @@
 # Ansible Builder
 
-Ansible Builder is a tool that automates the process of building and shipping a set of execution environments using the schemas and tooling defined in various Ansible Collections.
+Ansible Builder is a tool that automates the process of building execution environments using the schemas and tooling defined in various Ansible Collections and by the user.
+
+## Execution Environments
+
+Execution environments are container images intended to be used by Ansible executors.
+Specifically, `ansible-runner` can make use of these images.
```

Comment:
A link to the Github project URL might be good for `ansible-runner`?

Reply:
I second that!